README.md

```diff
@@ -51,14 +51,14 @@ For example, the implementation of a metric is not required to perform an expens
 calculating percentiles, if percentiles are not included in the set of values defined for this metric.           
 See [Getting Started](#getting-started) for details and usage examples.        
 
-### Advanced support for dimensional metrics
+### Advanced support for labeled (dimensional) metrics
```

Comment:
here and in the other place "dimensional" seems to be redundant (maybe, even misleading).
So I suggest to remove it completely.